package.json

```diff
@@ -1,11 +1,54 @@
 {
-  "name": "bpmn-js-example-data-extension",
+  "name": "@bpmn-io/bpmn-js-json-data-provider",
```

Comment:
:+1: 

Good to namespace an experimental (micro) utility.

Reply:
However I'm wondering if we could make more clear what it does:

* It is specific to Camunda, and adds example data facilities there.